jobs/build/operator-sdk_sync/Jenkinsfile

```diff
@@ -42,39 +47,62 @@ pipeline {
         stage('pre-flight') {
             steps {
                 script {
+                    if (!params.OCP_VERSION) {
+                        error 'OCP_VERSION must be specified'
+                    }
                     if (!params.BUILD_TAG) {
                         error 'BUILD_TAG must be specified'
                     }
-                    if (!params.VERSION) {
-                        error 'VERSION must be specified'
-                    }
-                    currentBuild.displayName = "${params.VERSION}"
-                    currentBuild.description = "${params.BUILD_TAG}"
+                    sdkVersion = ''
+                    buildvmArch = sh(script: 'arch', returnStdout: true).trim()
+                    manifestList = readJSON(
+                        text: sh(
+                            script: "skopeo inspect --raw docker://${imagePath}:${params.BUILD_TAG}",
+                            returnStdout: true,
+                        )
+                    ).manifests
                 }
             }
         }
         stage('extract binaries') {
             steps {
                 script {
-                    def manifestList = readJSON(
-                        text: sh(
-                            script: "skopeo inspect --raw docker://${imagePath}:${params.BUILD_TAG}",
-                            returnStdout: true,
-                        )
-                    ).manifests
                     manifestList.each {
-                        sh "rm -rf ./${params.VERSION} && mkdir ./${params.VERSION}"
+                        def sdkArch = it.platform.architecture == 'amd64' ? 'x86_64' : it.platform.architecture
```

Comment:
This won't work when aarch64 (arm64 in go-speak) is enabled.

Reply:
Good to know, we have that translation in multiple places :-/ We'll probably want a centralized go-2-RH translation method.

Reply:
I am looking at this @sosiouxme where should this go-2-RH method live? thinking groovy or python? Looking to help out here.